src/core/components/pin/pin-manager.js

```diff
@@ -51,30 +79,21 @@ class PinManager {
     return Array.from(this.recursivePins, key => new CID(key).buffer)
   }
 
-  getIndirectKeys (callback) {
+  getIndirectKeys ({ preload }, callback) {
```

Comment:
Do we want to preload when getting indirect keys?

Reply:
As implemented we leave it up to the caller and it's an undocumented option that defaults to false.

In general I think we want to be really careful with preloading because you can end up inadvertently sending huge amounts of data over the wire, maxing out the number of allowed connections, etc.